dev/io.openliberty.microprofile5.internal_fat/publish/servers/MP5CompatibleServer/MP50andEE8.xml

```diff
@@ -0,0 +1,9 @@
+<server description="Server for testing microProfile">
+
+    <include location="../fatTestPorts.xml"/>
+
+    <featureManager>
+        <feature>microProfile-5.0</feature>
```

Comment:
Do you have the similar compatibility tests for MP 4.1? e.g. MP41andEE9 should fail.

Reply:
no but at the moment MP 4.1 is using mpCompatible-4.0 so it can't be any different to the existing MP40andEE9 test. I could create a mpCompatible-4.1 but that seems like a lot of work for little to no gain.